instrumentation-core/spring/spring-webflux-5.0/spring-webflux-5.0.gradle

```
@@ -9,7 +9,7 @@ ext {
 
 apply from: "$rootDir/gradle/java.gradle"
 
-group = 'io.opentelemetry.instrumentation.spring.webflux'
+group = 'io.opentelemetry.instrumentation.spring'
```

I think let's not create maven subgroups, and put manual instrumentation directly in `io.opentelemetry.instrumentation` (which matches what we are doing in `io.opentelemetry.instrumentation.auto`)
```suggestion
group = 'io.opentelemetry.instrumentation'
```

---

I'll make this change in the spring-web and spring-webmvc projects as well